src/patternfly/components/Wizard/wizard.scss

```diff
@@ -1,23 +1,6 @@
 .pf-c-wizard {
-  --pf-c-wizard--BoxShadow: var(--pf-global--BoxShadow--xl);
   --pf-c-wizard--Height: 100%;
-  --pf-c-wizard--Width: 100vw;
-  --pf-c-wizard--lg--Width: calc(100% - (var(--pf-global--spacer--2xl) * 2));
-  --pf-c-wizard--lg--MaxWidth: #{pf-size-prem(1232px)}; // Shares with with about modal
-  --pf-c-wizard--lg--Height: #{pf-size-prem(762px)}; // Shares height with about modal
-  --pf-c-wizard--lg--MaxHeight: calc(100% - (var(--pf-global--spacer--2xl) * 2));
-  --pf-c-wizard--m-full-width--lg--MaxWidth: auto;
-  --pf-c-wizard--m-full-height--lg--Height: 100%;
-  --pf-c-wizard--m-in-page--BoxShadow: none;
-  --pf-c-wizard--m-in-page--Height: 100%;
-  --pf-c-wizard--m-in-page--Width: auto;
-  --pf-c-wizard--m-in-page--lg--MaxWidth: none;
-  --pf-c-wizard--m-in-page--lg--MaxHeight: none;
-
-  @media screen and (min-width: $pf-global--breakpoint--lg) {
-    --pf-c-wizard--Width: var(--pf-c-wizard--lg--Width);
-    --pf-c-wizard--Height: var(--pf-c-wizard--lg--Height);
-  }
+  --pf-c-modal-box--c-wizard--Height: #{pf-size-prem(762px)};
```

Comment:
this is such a specific height, maybe add a comment explaining why it is this height